### PR TITLE
completed html-syntax exercise - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-27 at 10 21 16 AM

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Syntax</title>
+</head>
+
+<body>
+  <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/006.png" />
```
Comment (shawnkost) [inline html-syntax/index.html]: Every `img` should have an `alt` attribute.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Bryan!

File: html-syntax/index.html (hunk 1 above)
Comment (bbrown949) [reply html-syntax/index.html]: added.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**